.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run lint
```

Comment:
Could we run the tests as well here?

```suggestion
npm run lint
npm test
npm test:e2e
```

Reply:
Yes, we can do this. But then we probably cant push any WIP changes (like step-definitions) for example to the PR. They have to be resolved locally then, or am I wrong?

Reply:
Ordinarily yes, though you can pass the `--no-verify` flag to skip the pre-commit hooks.

Reply:
Ordinarily yes, though you can pass the --no-verify flag to skip the pre-commit hooks.

Reply:
Added the scripts to the hook